frontend/src/components/MessagesPage.vue

```diff
@@ -1,83 +1,83 @@
-              :items-per-page="-1"
-              hide-default-footer
-              @click:row="rowClickHandler"
-            >
-              <template #no-data>
-                <v-alert :value="true" type="info"> No Data </v-alert>
-              </template>
-              <template #item.isRead="{ item }">
-                {{ item.isRead ? 'Read' : 'Unread' }}
-              </template>
-              <template #item.programYearValue="{ item }">
-                {{ formatFiscalYearName(item.programYearValue) }}
-              </template>
-            </v-data-table>
-          </v-card>
-        </v-col>
-        <v-col v-if="message.sender" id="messages-content" fluid class="pa-0" :cols="8">
-          <v-card class="pa-4 overflow-auto elevation-0" fluid tile :height="fitScreenHeight()">
-            <v-card-title class="pa-0">
-              <v-row>
-                <v-col :cols="8">
-                  {{ message.sender }}
-                </v-col>
-                <v-col align="right" :cols="4">
-                  {{ message.dateReceived }}
-                </v-col>
-              </v-row>
-              <v-row>
-                <v-col>
-                  <strong>{{ message.subject }}</strong>
-                </v-col>
-              </v-row>
-            </v-card-title>
-            <v-divider />
-            <v-card-text v-html="message.messageContent"></v-card-text>
-          </v-card>
-        </v-col>
-      </v-row>
+  <v-container id="messages" fluid class="pa-0">
+    <Spinner v-if="loading" />
+    <v-row v-else no-gutters>
+      <v-col id="messages-summary" cols="12" md="5" :class="borderClass">
+        <v-card tile elevation="0">
+          <v-data-table
+            :headers="columns"
+            :items="messages"
+            :mobile="null"
+            mobile-breakpoint="lg"
+            :height="screenHeight"
+            fixed-header
+            hover
+            item-key="messageId"
+            item-value="messageId"
+            :items-per-page="-1"
+            hide-default-footer
+            :row-props="getRowProps"
+            @click:row="rowClickHandler"
+          >
+            <template #no-data>
+              <v-alert :value="true" type="info"> No Data </v-alert>
+            </template>
+            <template #item.isRead="{ item }">
+              {{ item.isRead ? 'Read' : 'Unread' }}
+            </template>
+            <template #item.programYearValue="{ item }">
+              {{ formatFiscalYearName(item.programYearValue) }}
+            </template>
+          </v-data-table>
+        </v-card>
+      </v-col>
+      <v-col v-if="message.sender" id="messages-content" cols="12" md="7" class="mt-6 mt-md-0">
+        <v-card class="px-6 overflow-auto elevation-0" :height="screenHeight">
+          <div>
+            <v-row>
+              <v-col cols="12" sm="8">
+                {{ message.sender }}
+              </v-col>
+              <v-col align="right" cols="12" sm="4">
```

Comment:
Nitpick: the prop 'align' shows as an error from my linter. Can we change it to class="text-right" ?

Reply:
it's updated